source/includes/gcp/_target_proxies.md

```diff
@@ -64,14 +65,18 @@ curl -X GET \
 ```json
 {
     "data": {
-        "creationTimestamp": "2019-10-01T08:52:41.664-07:00",
-        "id": "2137054791002409126",
-        "kind": "compute#targetHttpProxy",
-        "name": "targetProxyName",
-        "selfLink": "https://www.googleapis.com/compute/v1/projects/testAreaName/global/targetHttpProxies/targetProxyName",
-        "urlMap": "https://www.googleapis.com/compute/v1/projects/testAreaName/global/urlMaps/urlMapName",
-        "type": "HTTP",
-        "shortUrlMap": "urlMapName"
+         "creationTimestamp": "2019-10-01T08:52:41.664-07:00",
+         "id": "2137054791002409126",
+         "kind": "compute#targetHttpsProxy",
+         "name": "targetProxyName",
+         "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area-oox/global/targetHttpsProxies/targetProxyName",
+         "sslCertificates": [
+           "https://www.googleapis.com/compute/v1/projects/test-area-oox/global/sslCertificates/sslCertificateName1"
+         ],
+         "type": "HTTPS",
+         "urlMap": "https://www.googleapis.com/compute/v1/projects/test-area-oox/global/urlMaps/urlMapName",
+         "shortUrlMap": "urlMapName",
+         "shortCertificates": "sslCertificateName"
```

Comment:
Sorry, the name of your cert above is `sslCertificateName1`. So change one of the other :-)

Reply:
3df058f313111b9807356745eb399f801937f674